libmscore/stafftype.cpp

```diff
@@ -1137,6 +1141,10 @@ bool TablatureDurationFont::read(XmlReader& e)
                         displayValue[int(TabVal::VAL_128)] = chr;
                   else if (val == "256")
                         displayValue[int(TabVal::VAL_256)] = chr;
+//                  else if (val == "512")
+//                        displayValue[int(TabVal::VAL_512)] = chr;
+//                  else if (val == "1024")
+//                        displayValue[int(TabVal::VAL_1024)] = chr;
```

Comment:
Why commented out?

Reply:
Because it is still WIP and `TabVal::VAL_512` and `TabVal::VAL_1024` don't exist (yet) ;-)

Reply:
There you go ;-)